docs/spikes/azure-sphere.md

```diff
@@ -0,0 +1,90 @@
+# What is Azure Sphere
+
+[Azure Sphere](https://azure.microsoft.com/en-us/services/azure-sphere/) is a security service
+combining hardware, OS-level software, and cloud services to secure IoT applications. Together these
+three components ensure [The Seven Properties of Highly Secure
+Devices](https://www.microsoft.com/en-us/research/uploads/prod/2020/11/Seven-Properties-of-Highly-Secured-Devices-2nd-Edition-R1.pdf).
+
+[Pricing](https://azure.microsoft.com/en-us/pricing/details/azure-sphere/): one time purchase of
+hardware only (no recurring fees)
+
+## Advantages for the starter kit
+
+- Automated management of the underlying OS and the concentrator logic.
```

Comment:
What do you mean by concentrator logic? Maybe a brief explanation would be helpful.

Reply:
I meant the LBS images. Reworded it to be more explicit.